lib/relay.js

```diff
@@ -55,8 +55,33 @@ function requestHandler(filterRules) {
 function responseHandler(filterRules, config) {
   const filters = Filters(filterRules);
 
+  const supportedFiles = (filterRules || [])
+    .filter(f => f.path && f.path.includes('/contents/'))
+    .map(f => f.path.split('/').pop());
```

Comment:
Let's move this into the `rewriteRawApi` function so that it is clear it is not used outside of its scope.

Reply:
It's a tradeoff of being run all the time, and running it one time.

Reply:
You are right!